conf/modules.config

```
@@ -11,17 +11,12 @@
 ----------------------------------------------------------------------------------------
 */
 
-// Generic process options for all workflows
+// CUSTOM_DUMPSOFTWAREVERSIONS
```

```suggestion
// Generic process options for all workflows
```

I think this comes from the template, so maybe we can leave it in like this